### ISSUE TITLE
Any thought on these loss graphs?

### ISSUE THREAD

**Issue by mark-joe** [affinelayer/pix2pix-tensorflow#116]
Did use the original PyTorch for a while but I needed 512x512 sized network, and find the adapations here. Now with TensorBoard, looking at the losses is easy, see below. What do you guys thinking of the graphs? So pix2pix adapted to 512x512, NO data augmentation in terms of cropping, my images are 512x512, so they go straight in.  Any thoughts are welcome! 

![loss](https://user-images.githubusercontent.com/38462062/38867164-84ae768c-4243-11e8-8e4d-df02c7cc0e2a.png)

**Comment by julien2512** [comment]
Hi,

I don't know about your dataset, but does 0.014 is enough for your work ? You should end your learning to 55000 steps to avoid the divergency.

It also seems the L1 gradient takes an other way while the GAN wins over the discriminator !
How much is your gan_weight over the L1_weight ?

Regards,
